src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The bye command, that terminates the program.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Constructs the bye command.
+     */
+    public ByeCommand() {
+        super("bye");
+    }
+
+    /**
+     * Executes the main logic of the bye command.

```

Comment:
I believe there should have empty line between description and parameter section 👍